### PR TITLE
adding ppanggolin version 2.1.2

### PR DESCRIPTION
**Pull request by erinyoung**
There's a new version of ppanggolin!

According to the [release notes](https://github.com/labgem/PPanGGOLiN/releases/tag/2.1.2) there are some new features and some bug fixes with this version.

I just copied the files from 2.0.5 and updated the version ARG.

The full diff:
```
 $ diff ppanggolin/2.0.5/Dockerfile ppanggolin/2.1.2/Dockerfile 
1c1
< ARG PPANGGOLIN_VER="2.0.5"
---
> ARG PPANGGOLIN_VER="2.1.2"
3c3
< FROM mambaorg/micromamba:1.5.6 as app
---
> FROM mambaorg/micromamba:1.5.8 as app
8c8
< LABEL base.image="mambaorg/micromamba:1.5.6"
---
> LABEL base.image="mambaorg/micromamba:1.5.8"
```
Pull Request (PR) checklist:
- [x] Include a description of what is in this pull request in this message.
- [x] The dockerfile successfully builds to a test target for the user creating the PR. (i.e. `docker build --tag samtools:1.15test --target test docker-builds/samtools/1.15` )
- [x] Directory structure as name of the tool in lower case with special characters removed with a subdirectory of the version number (i.e. `spades/3.12.0/Dockerfile`)
   - [x] (optional) All test files are located in same directory as the Dockerfile (i.e. `shigatyper/2.0.1/test.sh`)
- [x] Create a simple container-specific [README.md](https://github.com/StaPH-B/docker-builds/blob/master/.github/workflow-templates/readme-template.md) in the same directory as the Dockerfile (i.e. `spades/3.12.0/README.md`)
   - [x] If this README is longer than 30 lines, there is an explanation as to why more detail was needed
- [x] Dockerfile includes the recommended [LABELS](https://github.com/StaPH-B/docker-builds/blob/master/dockerfile-template/Dockerfile#L8-L18) 
- [x] Main [README.md](https://github.com/StaPH-B/docker-builds/blob/master/README.md) has been updated to include the tool and/or version of the dockerfile(s) in this PR
- [x] [Program_Licenses.md](https://github.com/StaPH-B/docker-builds/blob/master/Program_Licenses.md) contains the tool(s) used in this PR and has been updated for any missing

### CODE REVIEW

File: ppanggolin/2.1.2/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    wget && \
+    apt-get autoclean && rm -rf /var/lib/apt/lists/*
+
+ENV PATH="/opt/conda/envs/base/bin/:/opt/conda/bin/:$PATH" LC_ALL=C
+
+RUN micromamba install --name base -c conda-forge -c bioconda -c defaults ppanggolin=${PPANGGOLIN_VER} && \
+    micromamba clean -a -y && \
+    ppanggolin -h && \
+    mkdir /data
+
```
Comment (Kincekara) [inline ppanggolin/2.1.2/Dockerfile:32]: Please force package cache cleaning. 
I think lines 30 and 31 are also redundant here.

Comment (erinyoung) [reply ppanggolin/2.1.2/Dockerfile:30]: I added the -f flag and removed lines 30 & 31

Thank you for the suggestions!

**Comment by Kincekara** [comment]
Thank you for the changes! I am merging and deploying this image.

**Review by Kincekara — APPROVED** [review]

**Comment by Kincekara** [comment]
Thank you for updating this image! 
You can check the image deployment here: https://github.com/StaPH-B/docker-builds/actions/runs/11147841105

**Pull request merged**